packages/server/lambdas/alfresco/delete-folder.ts

```diff
@@ -76,9 +71,8 @@ export async function handleRequest(event: ALBEvent): Promise<ALBResult | undefi
     auditLog.info(user, `Deleted folder ${nodeId} in ${categoryData.alfrescoFolder}`);
 
     return {
-      statusCode: 200,
+      statusCode: 204,
```

Comment:
Hmm why not `statusCode: 200`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204